src/ostorlab/agent/mixins/agent_report_vulnerability_mixin.py

```diff
@@ -29,6 +29,7 @@ class MetadataType(enum.Enum):
     PORT = enum.auto()
     LOG = enum.auto()
     PACKAGE_NAME = enum.auto()
+    PACKAGE_VERSION = enum.auto()
```

Comment:
Should also be added  to the protobuff message definition.

Reply:
Added